docs-translations/uk-UA/tutorial/quick-start.md

```diff
@@ -0,0 +1,222 @@
+# Швидкий старт
+
+Electron дозволяє вам робити додатки на робочий стіл на чистому Javascript,
+через багатий API. Ви можете представляти його як Node.js додаток, який
+орієнтований для робочого стола, а не для веб сервера.
+
+Але це не означає, що Electron лиш привязка до GUI бібліотекам. На справді
+Electron використовує веб-сторінки як інтерфейс, так що його можна рахувати
+як невеличким Chroumium браузером, який контролюється за допомогою Javascript.
+
+### Головний процес
+
+В Electron процес, який запускає `main` з `package.json` називається
+__головним процемос__. Скрипт, який працює в головному процесі може показувати
+GUI, створюючи веб-сторінки
+
+### Процес-рендерер
+
+Так як Electron використовує Chromium для показу веб-сторінок,
+мульти-процесову архітектуру показу сторінок Chromium він теж використовує.
+Кожна веб-сторінка в Electron працює в своєму власному процесі,
+який називається __процес-рендерер__
+
+В звичайних браузерах веб-сторінки зазвичай запускаються в "пісочниці" і їм недоступні реальні ресурси комп'ютера. Користувачі Electron навпроти можуть використовувати API Node.js на сторінках, що використовує більш низькорівневу роботу з операційними системами.
+
+### Різниця між головним процесором і процесором-рендерером
+
+Головний процес створює веб-сторінки використовує `BrowserWindow`. Кожен
+екземпляр `BrowserWindow` показує веб сторінку через свій власний процесор-рендерер.
+Якщо екземпляр `BrowserWindow` знищується, то і відповідний процесор-рендерер теж
+завершується.
+
+Головний процес керує всіма веб-сторінками і відповідними їм процесами-рендерами.
+Кожний процес-рендерер ізольований і керує тільки своєю веб-сторінкою.
+
+На веб-сторінках не дозволяється викликати нативні API, які керують GUI,
+тому що це дуже небезпечно і може легко викликати витік ресурсів. Якщо ви хочете
+виконати дії з GUI на сторінці, процес-рендерер цієї сторінки повинен
+"Попросити" головний процес зробити ці дії.
+
+У Electron є декілька способів спілкування між процесам. Наприклад, модулі
+[`ipcRenderer`](https://electron.atom.io/docs/api/ipc-renderer/) и [`ipcMain`](https://electron.atom.io/docs/api/ipc-main/) використовується
```

Comment:
Replace to `../../../docs/api/ipc-renderer.md` and `../../../docs/api/ipc-main.md`